spa_ui/self_service/client/app/states/marketplace/details/details.state.js

```diff
@@ -40,11 +40,44 @@
   }
 
   /** @ngInject */
-  function StateController(dialogs, serviceTemplate) {
+  function StateController($state, CollectionsApi, dialogs, serviceTemplate, Notifications) {
     var vm = this;
 
     vm.title = 'Service Template Details';
     vm.dialogs = dialogs.resources[0].content;
     vm.serviceTemplate = serviceTemplate;
+    vm.submitDialog = submitDialog;
+
+    function submitDialog() {
+      var dialogFieldData = {
+        href: 'http://localhost:3000/api/service_templates/' + serviceTemplate.id
```

Comment:
@AllenBW @stackus Just realized I hardcoded this...what's the preferred way of getting the application hostname?